src/test/java/it/com/atlassian/shele/shelePlugin/MyComponentWiredTest.java

```diff
@@ -0,0 +1,28 @@
+package it.com.atlassian.shele.shelePlugin;
+
+import com.atlassian.plugins.osgi.test.AtlassianPluginsTestRunner;
+import com.atlassian.sal.api.ApplicationProperties;
+import com.atlassian.shele.shelePlugin.api.MyPluginComponent;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(AtlassianPluginsTestRunner.class)
+public class MyComponentWiredTest
```

Comment:
is it auto-generated test or just some example? in that case it does not have any value and could be removed

Reply:
auto-generated test